verification/curator-service/ui/src/components/CaseForm.tsx

```diff
@@ -796,6 +796,13 @@ class CaseForm extends React.Component<Props, CaseFormState> {
                                 </div>
                             </nav>
                             <div className={classes.form}>
+                                <Typography variant="h4">
+                                    Enter the details for a new case
```

Comment:
We should change this text (the new case part) depending on if it's the new case or edit case flow.

Reply:
sounds good, done.